mitosheet/mitosheet/api/get_code_snippets.py

```diff
@@ -63,4 +73,21 @@
 ]
 
 def get_code_snippets(params: Dict[str, Any], steps_manager: StepsManagerType) -> str:
-    return json.dumps(DEFAULT_CODE_SNIPPETS)
+        # If code snippet environment variables are set, then use them to get the code snippets. 
+        mito_config_code_snippets_version = os.environ.get(MITO_CONFIG_KEY_CODE_SNIPPETS_VERSION)
+        mito_config_code_snippets_url = os.environ.get(MITO_CONFIG_KEY_CODE_SNIPPETS_URL)
+        if mito_config_code_snippets_version == '1' and mito_config_code_snippets_url is not None:
+                
+                # Request the code snippets from the url
+                response = req.get(mito_config_code_snippets_url, verify=False)
+
+                if response.status_code == 200:
+                        # Parse the respone body into JSON 
+                        code_snippets = response.json()
+                        return json.dumps(code_snippets)
```

Comment:
We aren't doing the caching we said we're going to do in the spec. 

Let's pull out all this logic into a `get_custom_code_snippets` function and cache w/ a global variable the first time this function is run.